src/my-requests.tsx

```diff
@@ -208,114 +210,180 @@ const ProceedingRow = (props: ProceedingRowProps) => {
 
     const original_request = findOriginalRequest(props.proceeding);
 
+    const initialMessage = useMemo<Omit<Message, 'id'>>(
+        () => ({
+            transport_medium: 'email',
+            date: new Date(),
+            reference: props.proceeding.reference,
+            sentByMe: false,
+            correspondent_address: original_request?.correspondent_address || '',
+            correspondent_email: original_request?.correspondent_email || '',
+            type: 'response',
+        }),
+        [props.proceeding.reference, original_request]
+    );
+
+    const ImportModalContent = () => {
+        const [newMessage, setNewMessage] = useState<Omit<Message, 'id'>>(initialMessage);
+        const addMessage = useProceedingsStore((state) => state.addMessage);
+
+        return (
+            <>
+                <p>
+                    <Text id="import-message-modal-explanation" />
+                </p>
+                <MessageMetadataInput
+                    message={newMessage}
+                    onChange={(message) => setNewMessage(message)}
+                    includeContent={true}
+                />
+                <button
```

Comment:
Why aren't you using the proper modal buttons for this?

Reply:
Because it then rerendered the whole modal instead of just a part of it, so I decided to move the buttons in the content.

Reply:
But now the styling is inconsistent with all other modals. And I really don't like having to remember to change the styling here as well if we ever change the modal styling (as you said you planned to).